### PR TITLE
Add dicom2nifti

### PR DESCRIPTION
**Pull request by abrys**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/dicom2nifti/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+{% set name = "dicom2nifti" %}
+{% set version = "2.2.1" %}
+{% set sha256 = "b712e5ee2c367d12cd0a3139bcd604daf1da454bcc8da8b204882696c8c39b77" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
```
Comment (npavlovikj) [inline recipes/dicom2nifti/meta.yaml]: `fn` is an old syntax that is no longer used, so please remove that line.

File: recipes/dicom2nifti/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+    - setuptools
```
Comment (npavlovikj) [inline recipes/dicom2nifti/meta.yaml]: I don't think `setuptools` is needed here.

File: recipes/dicom2nifti/meta.yaml (hunk 1 above)
Comment (abrys) [reply recipes/dicom2nifti/meta.yaml]: Thanks for the review. The line was removed.

File: recipes/dicom2nifti/meta.yaml (hunk 2 above)
Comment (abrys) [reply recipes/dicom2nifti/meta.yaml]: Thanks for the review. The setuptools was removed.

**Comment by npavlovikj** [comment]
Thanks for the prompt fixes, and contributing, @abrys 🎉

**Pull request merged**